src/test/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCallerSuite.scala

```diff
@@ -21,188 +21,138 @@ class GermlineAssemblyCallerSuite extends FunSuite with Matchers with BeforeAndA
   var reference: ReferenceBroadcast = _
   var readSet: ReadSet = _
 
-  before {
+  override def beforeAll() {
```

Comment:
q: I had thought this was what the before was doing, but perhaps I misinterpreted, OOC what's the difference here? and the effect of the override?